scripts/init/vyos-router

```diff
@@ -278,6 +278,10 @@ start ()
     # all daemons if just one failed, do the start manually
     /usr/lib/frr/frrinit.sh start
 
+    # Delete container file network.
+    # Configuration should be loaded from VyOS cli.
+    rm -f /etc/cni/net.d/*
```

Comment:
Another idea would be to mount a temporary filesystem on that explicit folder

```bash
cni_dir="/etc/cni/net.d"
if [ ! -d ${cni_dir} ]; then
    mkdir -p ${cni_dir}
fi
mount -t tmpfs none ${cni_dir}
```

Reply:
> Another idea would be to mount a temporary filesystem on that explicit folder

Done